portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase

```diff
@@ -0,0 +1,526 @@
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/fields",
+				restEndpoint = "/",
+				restSchema = "FDSField");
+		}
+
+		task ("And add new view") {
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				dataSetViewDescription = "FDSViewDescription",
+				key_dataSetViewNameList = "View Test");
+		}
+
+		task ("When the user goes to the Action tab") {
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+		}
+
+		task ("When the user goes to the Action tab.") {
+			DataSetAdmin.goToTab(tabName = "Actions");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		DataSetAdmin.deleteAllDataSetEntries();
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-194351 Confirm that when editing an already created Link action, it is not possible to leave the fields empty"
+	@priority = 4
+	test CanAssertThatTheTypeFieldIsDisabled {
+		task ("And creates a new item action") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				type = "Link",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+		}
+
+		task ("When the user clicks on the new item action 3 dots button and edit") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			Click(
+				key_menuItem = "Edit",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("Then Error messages are displayed and the save button is disabled") {
+			AssertElementPresent(
+				key_placeholderText = "Please select an option.",
+				locator1 = "DataSet#ANY_DISABLED");
+		}
+	}
+
+	@description = "LPS-194351 confirm that when creating an action, and clicking on the three dots button, the "Edit" option is displayed"
+	@priority = 3
+	test CanAssertTheEditButtonAction {
+		task ("And creates a new item action") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				type = "Link",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+		}
+
+		task ("When the user clicks on the new item action 3 dots button") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			AssertElementPresent(
+				key_menuItem = "Edit",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("Then the Delete and Edit option is listed in the dropdown menu") {
+			AssertElementPresent(
+				key_menuItem = "Delete",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be deleted, and a success message is displayed"
+	@priority = 5
+	test CanBeDeleteAction {
```

Comment:
This is a priority 5 test, so you have to add the `property portal.acceptance = "true";`